packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -11,78 +11,110 @@
  */
 
 import {ActionButton} from '@react-spectrum/button';
+import {Flex} from '@react-spectrum/layout';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
 import {Tooltip, TooltipTrigger} from '../src';
 
 storiesOf('TooltipTrigger', module)
   .add(
     'default',
-    () => render('This is a tooltip.', {})
-  ).add(
+    () => render({})
+  )
+  .add(
     'placement: left',
-    () => render('This is a tooltip.', {placement: 'start'})
-  ).add(
+    () => render({placement: 'start'})
+  )
+  .add(
     'placement: top',
-    () => render('This is a tooltip.', {placement: 'top'})
-  ).add(
+    () => render({placement: 'top'})
+  )
+  .add(
     'placement: bottom',
-    () => render('This is a tooltip.', {placement: 'bottom'})
-  ).add(
+    () => render({placement: 'bottom'})
+  )
+  .add(
     'isDisabled',
-    () => render('This is a tooltip.', {placement: 'left', isDisabled: true})
-  ).add(
-     'multiple tooltips',
-     () => renderMultipleTriggers('This is a tooltip.', {placement: 'left'})
-   );
+    () => render({placement: 'left', isDisabled: true})
+  )
+  .add(
+    'delay',
+    () => render({delay: true})
+  )
+  .add(
+    'multiple tooltips',
+    () => renderMultipleTriggers({placement: 'left'})
+  )
+  .add(
+    'delay multiple tooltips',
+    () => renderMultipleTriggers({delay: true})
+  )
+  .add(
+    'isOpen',
+    () => renderMultipleTriggers({isOpen: true})
+  )
+  .add(
+    'trigger disabled',
+    () => renderDisabledTrigger()
+  );
 
-function render(content, props = {}) {
+function render(props = {}) {
   return (
     <TooltipTrigger {...props}>
       <ActionButton>Trigger Tooltip</ActionButton>
       <Tooltip>
-        {content}
+        Tooltip message.
+      </Tooltip>
+    </TooltipTrigger>
+  );
+}
+
+function renderDisabledTrigger() {
+  return (
+    <TooltipTrigger>
+      <ActionButton isDisabled>Trigger Tooltip</ActionButton>
+      <Tooltip>
+        Tooltip message.
```

Comment:
I really thought i'd written a comment about this somewhere. Here's the dilemma
If I disable the button, which is totally valid, I don't know about it in the trigger.
Should we disable all mouse events on disabled buttons/inputs?
`pointer-events: none;`

disabled makes it so that we can't focus it, so I think it makes sense that you shouldn't be able to hover it either?

It makes sense for a tooltiptrigger to be disabled but have the button still be interact-able, maybe the user has some setting to turn off tooltips so that they don't appear. At least, I think this makes sense.

Reply:
yeah I think that makes sense, to sum it up:
1. User can disable the TooltipTrigger which disables the tooltip from showing up on hover/focus (this doesn't cause the Button to be disabled)
2. User can disable the Button but not the TooltipTrigger. This causes the button to not receive any pointer events, preventing the hover tooltip from appearing (focus is already handled since the button is disabled)